### PR TITLE
ListView support and cleanup project

### PR DESCRIPTION
**Pull request by falkolab**
- changes to use this module in ListView
- update docs
- cleanup project
- .gitignore
- version 1.0.1

### CODE REVIEW

**Comment by GeraudBourdin** [comment]
hi thanks for this ! i do not have enough time to test it today. i ll try before the end of the week. 
But for what i've quickly seen, i didn't get what you've done to fix a listview issue ? 
i only see a line removing in ViewProxy.java. 

by the way thanks for contributing :)

File: android/src/com/geraudbourdin/svgview/ViewProxy.java [hunk 1]
```diff
@@ -67,7 +65,10 @@ public SvgView(TiViewProxy proxy) {
 			setNativeView(layout);
 		}
 
-		public void setImage() {
+		public void setImage() {			
+			if(image == null)
```
Comment (falkolab) [inline android/src/com/geraudbourdin/svgview/ViewProxy.java]: ListView do not sets properties while creating the View of proxy.

File: android/src/com/geraudbourdin/svgview/ViewProxy.java [hunk 2]
```diff
@@ -91,6 +92,10 @@ public void setImage() {
 		@Override
 		public void processProperties(KrollDict d) {
 			super.processProperties(d);
+			if (d.containsKey("image")) {
```
Comment (falkolab) [inline android/src/com/geraudbourdin/svgview/ViewProxy.java]: https://github.com/appcelerator/titanium_mobile/blob/master/android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java#L808

**Comment by falkolab** [comment]
I commented on the important places. You can test with this gist https://gist.github.com/falkolab/efa28be3c7f56e6d1f1f

**Comment by GeraudBourdin** [comment]
thanks again for this update!
I finally had time to test and update the repo. Feel free to send me comments if there is still something wrong on this release.

**Pull request merged**